app-i18n/man-pages-pl/man-pages-pl-0.7.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A collection of Polish translations of Linux manual pages"
+HOMEPAGE="https://sourceforge.net/projects/manpages-pl/"
+SRC_URI="mirror://sourceforge/manpages-pl/manpages-pl-${PV}.tar.bz2"
```

Comment:
It is customary to add something like:

    MY_P=manpages-pl-${PV}

to avoid having to repeat this in `SRC_URI` and `S`; but I'm leaving this up to you.

Reply:
Unfortunately it's manpages-pl in the archive, and man-pages-pl in the package name. If you have a good suggestion on how to resolve this, i'm open to suggestions.

I'll get to the rest of the issues atm.

Reply:
I did. Declare a custom variable like I suggested above, and use it instead of ${P}.

Reply:
ok, i got confused there for a moment, but i've fixed it.